tests/integration_tests/config/generate_default_metadata_to_yml_file.yml

```diff
@@ -1,6 +1,9 @@
 CTA_SIMPIPE:
   APPLICATION: simtools-generate-default-metadata
-  TEST_NAME: test
+  TEST_NAME: test-yaml
   CONFIGURATION:
     OUTPUT_FILE: test.yml
     OUTPUT_PATH: simtools-tests
+  INTEGRATION_TESTS:
+    - OUTPUT_FILE: test.yml
+    - FILE_TYPE: yml
```

Comment:
I was thinking that the file type test is redundant because of the suffix already being present in the output_file test, but after looking into `test_applications_from_config.py` I understand the importance of this test. However, `file_type` is actually testing whether opening the output file works (not only the type of the output). So, I suggest changing the name to `OPEN_OUTPUT_FILE` for example.

Reply:
The important point for yaml and json is that it uses the corresponding libraries to open it. So if the file is not of type yaml/json, the tests fails. I therefore think INTEGRATION_TEST:FILE_TYPE is a good name. For opening a file, one could use a very generic file-open, which does not knows anything about the type. 

Let me know if this makes sense.

Reply:
Ok, it is just the name and I don't have a strong opinion on that. It would be nice in the future (as you commented below) if we could test opening all kinds of output file in this test, but obviously not on this PR.